### PR TITLE
GH OIDC for deployment to strides-ampad-workflows account

### PR DESCRIPTION
**Pull request by zaro0508**
Commit bdfc556 had setup github OIDC access for the Sage-Bionetworks-Workflows/nextflow-infra repo.  However that setup didn't work because the template used was not adequate and needed to be reverted in commit f6de736

Now we need to redo the setup for strides-ampad-workflows with the help of PR https://github.com/Sage-Bionetworks/aws-infra/pull/394

depends on https://github.com/Sage-Bionetworks/aws-infra/pull/394

### CODE REVIEW

File: sceptre/strides-ampad-workflows/config/prod/github-oidc-provider.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.5/templates/IAM/oidc-provider.j2
```
Comment (xschildw) [inline sceptre/strides-ampad-workflows/config/prod/github-oidc-provider.yaml:3]: This one .yaml

**Review by xschildw — CHANGES_REQUESTED** [review]
Don't we need a dependency on oidc_provider.yaml to make sure we don't use it b4 it's setup (or does the !stack_output do it)?

**Comment by zaro0508** [comment]
> Don't we need a dependency on oidc_provider.yaml to make sure we don't use it b4 it's setup (or does the !stack_output do it)?

`!stack_output` does setup a dependency between the two files automatically while `!stack_ouput_external` does not

**Review by xschildw — CHANGES_REQUESTED** [review]
the ref to https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.5/templates/IAM/oidc-provider.j2 should be https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.5/templates/IAM/oidc-provider.yaml (line 3 in sceptre/strides-ampad-workflows/config/prod/github-oidc-provider.yaml).

**Review by xschildw — APPROVED** [review]

**Review by xschildw — APPROVED** [review]
Why does not stack_output work?

**Pull request merged**